CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 add_compile_options(-Wall -Wextra -Werror)
 add_compile_options(-Wcast-align -Wformat-nonliteral -Wmissing-format-attribute -Wredundant-decls -Wsign-compare -Wtype-limits -Wuninitialized -Wwrite-strings)
 add_compile_options(-Werror=unused-result -Wodr)
+add_compile_options(-Wno-nonnull)
```

Comment:
It would probably be obvious if I tried to build, but it would also be helpful if you included some reasoning in the commit message.

Reply:
We should ideally fix the issue rather than merging the PR with this commit.
I just wanted the commit to be buildable so anyone can try building and installing the RPMs.